modules/async_worker.py

```diff
@@ -376,11 +377,15 @@ def handler(async_task):
 
             progressbar(async_task, 3, 'Processing prompts ...')
             tasks = []
+            task_rng = random.Random(seed) # may bind to inpaint noise in the future
```

Comment:
@flannerybh why has the task_rng calculation been moved outside the for loop?

Reply:
I don't remember exactly what I was thinking there. Your did on my fork looks reasonable. I'm busy now but I'll look back and see if there was a problem another time.  What exactly was the problem with moving it outside? I was still seeing different words before when using multiple images.

Reply:
Let's imagine the following scenario:
- image number: 2
- prompt: `[[bird, elephant]], __color__`
- seed: 1234

![Screenshot 2024-02-25 at 21 56 56](https://github.com/lllyasviel/Fooocus/assets/9307310/c72a498f-06ea-4203-812b-b9bbdf05148a)


this will differ from the following prompt & config:
- image number: 1
- prompt: `elephant, __color__`
- seed: 1235

![Screenshot 2024-02-25 at 21 58 23](https://github.com/lllyasviel/Fooocus/assets/9307310/a5784258-742a-4a7a-9c9c-23d0c3f3afbe)

The issue is that the images differ when generating an image with the same task_rng as the wildcard changes in a different manner (both times with seed 1234) than when using a separate seed for each task.
When generating only one image the task_rng seed is set correctly (1235) and you're getting a different result.

When using `task_rng = random.Random(task_seed)` this can be prevented:

![Screenshot 2024-02-25 at 22 01 38](https://github.com/lllyasviel/Fooocus/assets/9307310/d1e58cd0-9c71-4fb6-a2a5-16395d90b07c)

Reply:
I remember now, the intention was to be able to freeze the seed, but still get different values from the wildcard. I understands the inconsistency though